src/main/resources/application.yml

```diff
@@ -30,3 +30,22 @@ ams:
 
 interop-party-registration:
   enabled: false
+
+spring:
+  profiles:
+    active: "bb,fin12"
+  jackson:
+    date-format: org.mifos.connector.fineractstub.RFC3339DateFormat
+    serialization:
+      WRITE_DATES_AS_TIMESTAMPS: false
+
+
+springfox:
```

Comment:
We are using spring docs for other services, can you tell me why spring fox needs to be adopted here?

Reply:
Tried with springdocs but getting import issues.